File: MPP/MPP Proiect/CSharp/model/Participant.cs
using System;

namespace model{
    
    [Serializable]
    public class Participant: Entity<int>{
        private string name;

        private int totalPoints;

        public string Name{
            set{ name = value; }
            get{ return name; }
        }

        public int TotalPoints{
            set{ totalPoints = value; }
            get{ return totalPoints; }
        }

        public Participant(string name, int totalPoints){
            this.name = name;
            this.totalPoints = totalPoints;
        }

        public override string ToString(){
            return "Name: " + Name + "Total Points: " + TotalPoints;
        }
    }
}

File: MPP/MPP Proiect/Java/Model/src/main/java/model/Participant.java
package model;

import java.io.Serializable;

public class Participant extends Entity<Integer> implements Serializable{

    private String name;

    private int totalPoints;

    public Participant(String name, int totalPoints) {
        this.name = name;
        this.totalPoints = totalPoints;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getTotalPoints() {
        return totalPoints;
    }

    public void setTotalPoints(int totalPoints) {
        this.totalPoints = totalPoints;
    }

    @Override
    public String toString() {
        return "Name: " + this.name + " Points: " + this.totalPoints;
    }
}
